src/mqt/bench/benchmark_generation.py

```diff
@@ -284,14 +284,16 @@ def get_native_gates_level(
     if file_precheck and path.is_file():
         return True
 
-    if gateset.name == "clifford+t":
+    if gateset.description == "clifford+t":
         from qiskit.transpiler import PassManager  # noqa: PLC0415
         from qiskit.transpiler.passes.synthesis import SolovayKitaev  # noqa: PLC0415
 
         # Transpile the circuit to single- and two-qubit gates including rotations
+        gates = {inst.name for inst, _ in gateset.instructions}
+        print(gates)
```

Comment:
There is a print statement here, which I am pretty surprised that ruff did not complain about or optimize away. Maybe worth checking the config.

Other than that, we should simply be able to use `*target.operation_names` in the call down below.

Reply:
Thanks for catching that, your suggestions works well.